opentelemetry-sdk/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- Add debug Controller and set initial checkpoint timestamp in aggregators
```

Comment:
Should be updated to remove the debug controller

Reply:
right, fixing...